test_ffi.pl

```diff
@@ -319,3 +320,76 @@
     ffi_findall_bug_(ffi_p(_), L).
 
 :- end_tests(call).
+
```

Comment:
This should go into src/Tests/core/test_text.pl

Reply:
Done